src/embedded.ts

```diff
@@ -248,7 +248,20 @@ export class RoomWidgetClient extends MatrixClient {
         throw new Error(`Unknown room: ${roomIdOrAlias}`);
     }
 
-    protected async encryptAndSendEvent(room: Room, event: MatrixEvent): Promise<ISendEventResponse> {
+    protected async encryptAndSendEvent(room: Room, event: MatrixEvent): Promise<ISendEventResponse>;
+    protected async encryptAndSendEvent<F extends ISendFutureRequestOpts>(
+        room: Room,
+        event: MatrixEvent,
+        futureOpts: F,
+    ): Promise<ISendFutureResponse<F>>;
+    protected async encryptAndSendEvent<F extends ISendFutureRequestOpts>(
+        room: Room,
+        event: MatrixEvent,
+        futureOpts?: F,
+    ): Promise<ISendEventResponse | ISendFutureResponse<F>> {
+        if (futureOpts) {
+            throw new Error("Future sending via widgets is not implemented");
```

Comment:
this is replaced in the widget api impl PR i guess?

Reply:
Yes it is!